package.json

```diff
@@ -19,6 +21,7 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "classnames": "2.3.1",
+    "date-fns": "^2.28.0",
     "flutterwave-react-v3": "1.2.0",
     "history": "5.3.0",
     "lodash.clonedeep": "4.5.0",
```

Comment:
Can you remove the `react-calendar` npm package too, and re-run yarn install to re-generate yarn.lock? Then push up the changes.